app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,182 @@
+    <div class="ui equal width stackable center aligned page grid">
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="country">{{t 'Country'}}</label>
+        </div>
+        <div class="left floated left aligned column">
+          <div class="ui fluid search selection dropdown">
+            <input type="hidden" name="country">
+            <i class="dropdown icon"></i>
+            <div class="default text">{{t 'Select Country'}}</div>
+            <div class="menu">
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="address">{{t 'Address'}}</label>
+        </div>
+        <div class="column">
+          <textarea rows="2"></textarea>
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="city">{{t 'City'}}</label>
+        </div>
+        <div class="column">
+          {{input type='text' id='city'}}
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="state">{{t 'State/Province'}}</label>
+        </div>
+        <div class="column">
+          {{input type='text' id='state'}}
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="zipcode">{{t 'Zip Code'}}</label>
+        </div>
+        <div class="left floated left aligned column">
+          {{input type='text' id='zipcode'}}
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="payvia">{{t 'Pay Via'}}</label>
+        </div>
+        <div class="left floated left aligned column">
+          <div class="grouped fields">
+            <div class="field">
+              <div class="ui radio checkbox">
+                <input type="radio" name="payment-stripe" checked="checked">
+                <label><img class="ui medium image" src="/images/payment-logos/stripe.png" alt="stripe"></label>
+              </div>
+            </div>
+            <div class="field">
+              <div class="ui radio checkbox">
+                <input type="radio" name="payment-paypal">
+                <label><img src="/images/payment-logos/paypal.png" alt="paypal"></label>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="column"></div>
+        <div class="left floated left aligned column">
+          {{t 'By clicking \"Pay Now\", I acknowledge that I have read and agree with the Open Event terms of services and privacy policy.'}}
```

Comment:
@harshitagupta30 Why are you escaping double quotes inside single quotes, they will be fine without the escape chatacter.